MetadataProcessor.Shared/nanoSkeletonGenerator.cs

```diff
@@ -190,10 +190,10 @@ private void GenerateStubs()
                                             parameterDeclaration.Type = parameterType;
 
                                             parameterDeclaration.Declaration = 
-                                                $"{parameterType} {parameterDeclaration.Name};" + Environment.NewLine +
-                                                $"        UINT8 heapblock{parameterIndex.ToString()}[CLR_RT_HEAP_BLOCK_SIZE];";
+                                                $"{parameterType} {parameterDeclaration.Name} = 0x0;" + Environment.NewLine +
+                                                $"        uint8_t heapblock{parameterIndex}[CLR_RT_HEAP_BLOCK_SIZE];";
 
-                                            parameterDeclaration.MarshallingDeclaration = $"Interop_Marshal_{parameterTypeClr}_ByRef( stack, heapblock{(parameterIndex + (m.IsStatic ? 0 : 1)).ToString()}, {parameterDeclaration.Name} )";
+                                            parameterDeclaration.MarshallingDeclaration = $"Interop_Marshal_{parameterTypeClr}_ByRef( stack, heapblock{(parameterIndex /*+ (m.IsStatic ? 0 : 1)*/ )}, {parameterDeclaration.Name} )";
```

Comment:
Why is the code `m.IsStatic` commented?
If it's not required anymore please remove.

Reply:
I commented it out as i did not know if I should fully remove it.
If I should remove it I will remove it for the rest as well as this probably only leads to errors.